datafusion/core/src/execution/session_state.rs

```diff
@@ -237,6 +248,33 @@ impl SessionState {
             function_factory: None,
         };
 
+        #[cfg(feature = "parquet")]
+        match new_self.register_file_format(Arc::new(ParquetFormatFactory::new()), false)
+        {
+            Ok(_) => (),
+            Err(e) => log::info!("Unable to register default ParquetFormat: {e}"),
+        };
```

Comment:
Another way to write this would be

```suggestion
        if let Err(e) = new_self.register_file_format(Arc::new(ParquetFormatFactory::new()), false)
        {
           log::info!("Unable to register default ParquetFormat: {e}"),
        };
```

which might be more concise 🤔